Automattic/wp-calypso
# Help Center: use support interactions

## Proposed Changes

1. Rewrite the odie context to remove unused props and simplify logic.
2. Removed the useQuery cache management in favor of just refetching.
3. Initialize a new support interaction when help center loads.
4. Reorganized the directory structure to be more appropriate.

## Why are these changes being made?

This unifies odie, zendesk, and help-center by adding state management. We will be storing the state in the support interaction database table and the local version is managed by a redux store. By updating the support interaction within the store you change what chats you see.

The PR got quite large because I added a few files and touched a lot of things organizing it to be easier to read and manage.

## Testing Instructions

* Checkout branch or use live link
* Navigate to `/home?flags=help-center-experience` and open the Help Center.
* Test out Help Center. Everything should work the same.
